docs/projects.md

```diff
@@ -25,7 +25,7 @@
     {
         "name": "Courses",
         "url": "/courses",
-        "imageUrl": "/static/courses/csintro3.gif"
+        "imageUrl": "/static/courses/csintro1.gif"
```

Comment:
What's with these changes?

Reply:
Our build automatically picks up a gif and regenerates the "projects.md" page.